fedimint-cli/src/lib.rs

```diff
@@ -631,7 +631,7 @@ impl FedimintCli {
                         .map_err_cli_general()?,
                     None => ws_api
                         .request_with_strategy(
-                            EventuallyConsistent::new(ws_api.peers().len()),
+                            CurrentConsensus::full_participation(ws_api.peers().len()),
```

Comment:
I think the problem lies here: the old eventually consistent afaik retried on errors as long as some good responses came back. This test has been flaky before, it's likely that we relied on some implicit retry behavior. Will try to replicate.

Reply:
Yep, looks if if that was the cause. I pushed the fix.

Reply:
Ok, *now* it should work.